rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -99,16 +102,19 @@
               tempest = load 'pipeline_steps/tempest.groovy'
               holland = load 'pipeline_steps/holland.groovy'
               maas = load 'pipeline_steps/maas.groovy'
+              kibana = load 'pipeline_steps/kibana.groovy'
           }}
           dir("rpc-gating/playbooks") {{
             writeFile file: "inventory/hosts", text: "[job_nodes]\n${{INSTANCE_NAME}} ansible_host=${{INSTANCE_IP}}\n"
             common.install_ansible()
           }}
           ssh_slave.connect()
           node(INSTANCE_NAME) {{
-            sh """
-               lvcreate -n aio -L250G lxc
-            """
+            if (env.STAGES.contains("Prepare LEM AIO")) {{
```

Comment:
The other uses of this seem to be when a stage has been created with `stage()`. Is this a "real" stage? I assume that means it won't appear in the table for the job as a stage. It seems strange that `STAGES` can include a stage that effectively doesn't exist. I'm not really expressing an opinion here, just trying to understand things better.